client/src/modules/dashboard/Sponsors/pages/SponsorPage.tsx

```diff
@@ -43,14 +44,20 @@ export const SponsorPage: NextPage = () => {
           Details{' '}
         </Heading>
         <Flex mt="2" justifyContent="space-between">
-          <Text data-cy="type">Type: {data?.sponsor?.type}</Text>
+          <Text data-cy="type">Type: {data?.sponsorWithEvents.type}</Text>
           <Text data-cy="website">
-            Website: <Link>{data?.sponsor?.website}</Link>
+            Website: <Link>{data?.sponsorWithEvents.website}</Link>
           </Text>
         </Flex>
       </Card>
-
-      <h3>Placeholder for events ....</h3>
+      {data.sponsorWithEvents && (
+        <EventsList
+          title={'Sponser'}
+          events={data.sponsorWithEvents.event_sponsors
+            .map((e) => e.events)
+            .flat()}
+        />
+      )}
```

Comment:
this is a workaround typescript error, there are more info in discord

https://discord.com/channels/692816967895220344/1024341059989409844/1024343219682689146